packages/gatsby-theme-carbon/src/components/MiniCard/MiniCard.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { Link } from 'gatsby';
+import {
+  Launch20,
+  Download20,
+  ArrowRight20,
+  Calendar20,
+  Email20,
+} from '@carbon/icons-react';
+import cx from 'classnames';
+import { settings } from 'carbon-components';
+import styles from './MiniCard.module.scss';
+
+const { prefix } = settings;
+
+const MiniCard = ({
+  children,
+  className,
+  href,
+  title,
+  actionIcon = 'launch',
+  ...rest
+}) => {
+  const icons = (
+    <>
+      {actionIcon === 'launch' ? <Launch20 aria-label="Open resource" /> : null}
+      {actionIcon === 'arrowRight' ? (
+        <ArrowRight20 aria-label="Open resource" />
+      ) : null}
+      {actionIcon === 'download' ? <Download20 aria-label="Download" /> : null}
+      {actionIcon === 'email' ? <Email20 aria-label="Email" /> : null}
+      {actionIcon === 'calendar' ? <Calendar20 aria-label="Calendar" /> : null}
+    </>
+  );
```

Comment:
I think the clearest way to do this would be with a `getIcon` helper function outside of the MiniCard that uses a switch statement to return the correct icon:

```suggestion
const getIcon = ({actionIcon}) => {
  switch (actionIcon) {
    case 'arrowRight':
      return <ArrowRight20 aria-label="Open resource" />
    case 'download':
        return <Download20 aria-label="Download" />;
    case 'email':
        return <Email20 aria-label="Email" />;
    case 'calendar':
        return <Calendar20 aria-label="Calendar" />;
    default:
        return <Launch20 aria-label="Open resource" />;
  }
}
```